src/Raven.Studio/wwwroot/App/views/manage/addClusterNode.html

```diff
@@ -14,7 +14,15 @@
                             <label for="url" class="control-label">URL</label>
                             <div class="flex-grow">
                                 <div data-bind="validationElement: serverUrl">
-                                    <input autocomplete="off" type="text" class="form-control" data-bind="textInput: serverUrl" id="url" placeholder="Enter server url"> 
+                                    <input autocomplete="off" type="text" class="form-control" data-bind="textInput: serverUrl" id="url" placeholder="Enter server url">
+                                </div>
+                            </div>
+                        </div>
+                        <div class="form-group">
+                            <label for="url" class="control-label">TAG</label>
```

Comment:
Can we replace TAG-> Tag + changelabel for url - since it is copy & paste issue.

Reply:
Do we need validation here? Or tag can be any string?

Reply:
Fixed

Reply:
I added server side validation.

Reply:
Can we add a simple JS validation here for better UX?

Reply:
Done